repos/cms_sw/cms_docker/githublabels.py

```diff
@@ -9,16 +9,10 @@
 
 for arch in ["x86_64", "ppc64le", "aarch64"]:
     TYPE_COMMANDS[arch] = [LABEL_COLORS["doc"], "%s-[0-9a-f]+" % arch, "mtype", True]
-    for status in [
-        ("build-queued", "pending"),
-        ("building", "pending"),
-        ("build-done", "approved"),
-        ("build-error", "rejected"),
-    ]:
-        arch_status = "%s-%s" % (arch, status[0])
-        TYPE_COMMANDS[arch_status] = [
-            LABEL_COLORS[status[1]],
-            "%s[-a-z]*%s" % (arch, status[0]),
-            "state",
-            True,
-        ]
+    TYPE_COMMANDS[arch + "-cms-docker"] = [
+        LABEL_COLORS["doc"],
+        "%s-[a-z]+-(queued|building|done|error)" % arch,
```

Comment:
@aandvalenzuela ,  this regexp should match `arch-something-(.*)` but what about `x86_64` tag itself

Reply:
@aandvalenzuela , I would suggest to change this to `"%s-([a-z]+-|)(queued|building|done|error)" % arch`